traitsui/qt4/helper.py

```diff
@@ -242,7 +242,7 @@ def wrap_text_with_elision(text, font, width, height):
             line = text_layout.createLine()
             if not line.isValid():
                 break
-            line.setLineWidth(width)
+            line.setLineWidth(int(width))
```

Comment:
The docstring claims that the `width` should be `int`. So if this fails it means that the code that calls wrap_text_with_elision is passing the wrong type. We might want to update the docstring if we make this change

Reply:
I'm comfortable with leaving it as is for defensive programming purposes.  This code that could be being called by 3rd party renderers, because it's used in code intended as an example.  So we may have users who have been passing floats and having things work, I'd rather have this continue, but keep the public API as int.  We can tighten this in a major release if we want to.